server/src/web/broadcast.rs

```diff
@@ -0,0 +1,137 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::time::Duration;
+use chrono::Utc;
+use futures::future::join_all;
+use log::debug;
+use sqlx::{Pool, Sqlite};
+use tokio::sync::Mutex;
+use actix_web_lab::sse;
+use actix_web_lab::sse::Sse;
+use actix_web_lab::util::InfallibleStream;
+use tokio_stream::wrappers::ReceiverStream;
+
+use crate::build::BuildSummary;
+
+const BUILD_START_EVENT: &str = "build_start";
+const BUILD_END_EVENT: &str = "build_end";
+const LOG_EVENT: &str = "log";
+const PING_EVENT: &str = "ping"; 
```

Comment:
These constants should be moved to the data module, so they can be reused in the cli and easily be changed. Also could they be implemented as an enum with a from- and tostring trait?